Remora.Rest/Json/DataObjectConverter.cs

```diff
@@ -733,6 +576,39 @@ private string GetWriteJsonPropertyName(PropertyInfo dtoProperty, JsonSerializer
         return createdConverter;
     }
 
+    /// <summary>
+    /// Gets the default value for a type or <see langword="null"/> if it has no default value.
+    /// </summary>
+    /// <remarks>
+    /// This always either <see langword="default"/> for <see cref="Optional{TValue}"/> or <see langword="null"/> for any other type.
+    /// </remarks>
+    /// <param name="type">The type to get the default value for.</param>
+    /// <returns>The default value or <see langword="null"/>.</returns>
+    internal object? GetDefaultValueForType(Type type)
+    {
+        return _dtoEmptyOptionals.GetValueOrDefault(type);
```

Comment:
A comment explaining why `_dtoEmptyOptionals` is used for getting the default value for any type would help here - it's not immediately obvious to someone unfamiliar with the code.

Reply:
https://github.com/Remora/Remora.Rest/blob/cfc35dafd76ba3732406714c94685f4fb712e9c1/Remora.Rest/Json/DataObjectConverter.cs#L589-L591

Reply:
That looks good. Slight grammatical error - `this method needs to be return null` => `this method returns null`.

Reply:
https://github.com/Remora/Remora.Rest/blob/ad8b62c0a88fd88efd6b222d065978e4b935e0f2/Remora.Rest/Json/DataObjectConverter.cs#L589-L591
My bad, fixed.

Reply:
:+1: